valkey.conf

```diff
@@ -1325,7 +1325,20 @@ lazyfree-lazy-user-flush no
 # to thread the write and read syscall and transfer the client buffers to the
 # socket and to enable threading of reads and protocol parsing. 
 #
-# NOTE 2: If you want to test the server speedup using valkey-benchmark, make
+# prefetch-batch-max-size 16
```

Comment:
The default value of the config normally goes at the end of the description, so move this line lower.

Reply:
Done